tests/integration/vpp/030_arp_test.go

```diff
@@ -85,17 +85,39 @@ func TestArp(t *testing.T) {
 			if err != nil {
 				t.Fatalf("adding arpentry failed: %v", err)
 			}
-			t.Logf("arpentry added %v", test.newArpEntry)
+			t.Logf("arpentry added %+v", test.newArpEntry)
 
 			arpentries, err = h.DumpArpEntries()
 			if err != nil {
 				t.Fatalf("dumping arpentries failed: %v", err)
 			}
-			arpentriescnt = len(arpentries)
-			t.Logf("%d arpentries dumped", arpentriescnt)
+			arpentriescnt2 := len(arpentries)
+			t.Logf("%d arpentries dumped", arpentriescnt2)
 
+			if arpentriescnt+1 != arpentriescnt2 {
+				t.Errorf("Number of arp entries after adding of one arp entry is not incremented by 1")
+			}
+
+			newArpEntryIsPresent := false
 			for _, arpentry := range arpentries {
-				t.Logf(" - arpentry: %+v", arpentry)
+				if (arpentry.Arp.Interface == test.newArpEntry.Interface) && (arpentry.Arp.IpAddress == test.newArpEntry.IpAddress) && (strings.ToLower(arpentry.Arp.PhysAddress) == strings.ToLower(test.newArpEntry.PhysAddress)) {
+					if test.newArpEntry.Static {
+						if arpentry.Arp.Static {
+							newArpEntryIsPresent = true
+							break
+						}
```

Comment:
I made this because arp dump does not return the field Static in case that it is false.
I am not sure if it is not too complicated... Or what happens if in returned structure is not some field present and I suppose that it exists ... I am not sure what is GoLang reaction to this ...

Reply:
Let's ignore the static field to simplify the test.

Reply:
ok